docs/building-a-node-with-ldk/setting-up-a-channel-manager.md

```diff
@@ -0,0 +1,1276 @@
+# Setting up a ChannelManager
+
+The ChannelManager is responsible for several tasks related to managing channel state. This includes keeping track of many channels, sending messages to appropriate channels, creating channels and more.
+
+## Adding a ChannelManager
+
+To add a `ChannelManager` to your application, run:
+
+<CodeSwitcher :languages="{rust:'Rust', kotlin:'Kotlin', swift:'Swift'}">
+  <template v-slot:rust>
+
+```rust
+use lightning::ln::channelmanager;
+
+let channel_manager = ChannelManager::new(
+  &fee_estimator,
+  &chain_monitor,
+  &broadcaster,
+  &router,
+  &logger,
+  &entropy_source,
+  &node_signer,
+  &signer_provider,
+  user_config,
+  chain_params,
+  current_timestamp
+);
+```
```

Comment:
This alone isn't very valuable and requires that prerequisites that follow. Additionally, it is repeated in greater detail later on under "Initialize the `ChannelManager`".

Reply:
Because the `ChannelManager` is so "involved" I find it useful to start with what one might end up with as it helps shape the mental model of a critical piece of the API. I feel like when you see this the next logical question is well how do I set up a `FeeEstimator` etc arguably it's already covered in the intro. Slight rewording https://github.com/lightningdevkit/lightningdevkit.org/pull/156/commits/d26a8e8d0f6ddb667f72cf2615c385f474217ed3